packages/lab/src/stepped-tracker/SteppedTracker.css

```diff
@@ -1,17 +1,41 @@
 .saltSteppedTracker {
-  margin: 0;
-  padding: 0;
-  text-indent: 0;
-  list-style-type: none;
-  display: flex;
+  grid-area: SteppedTracker;
   width: 100%;
-  position: relative;
+  height: 100%;
+
+  padding: var(--salt-focused-outlineWidth);
```

Comment:
I know this is here because of overflow hidden, is that absolutely necessary? compare to Accordion animation

Reply:
In Accordion you guys have used negative offset for the focus outline on the button part, which is not a bad solution. Don't know if it will pass with our designers and a11y experts, will check with them.